src/Helpers/CVEHelper.js

```diff
@@ -63,18 +63,18 @@ export function createSystemsListTable(affectedSystems) {
 export function createCveDetailsPage(cves) {
     let isLoading = cves.isLoading;
     let error = cves.error;
-
     if (!isLoading && cves?.payload?.data) {
         let payload = cves.payload.asMutable();
         let { data, meta } = payload;
-
+        let  clusterData  = cves?.clusterDetails?.payload.data;
         data = {
             impact: data.attributes.impact,
             public_date: processDate(data.attributes.public_date),
             description: data.attributes.description,
             synopsis: data.attributes.synopsis,
             rh_link: createRHDBLink(data.attributes.synopsis, messages.RHDBLink),
             mitre_link: createMitreLink(data.attributes.synopsis),
+            cluster_link: createClusterLink(clusterData?.length, data.id, messages.clusterLink),
```

Comment:
using `clusterData?.length` is not correct because you will fetch only the first page of results